blocks/extensions/monitoring.py

```diff
@@ -139,3 +140,17 @@ def do(self, callback_name, *args):
             self.add_records(self.main_loop.log,
                              self._buffer.get_aggregated_values().items())
             self._buffer.initialize_aggregators()
+
+
+class CrossEntropy(MonitoredQuantity):
```

Comment:
If this is going to be used for test purposes only, move it to the test file please.